errors.go

```diff
@@ -55,12 +55,16 @@ func NewErrorFromPanic(err interface{}) *Error {
 	if err != nil {
 		description = fmt.Sprint(err)
 	}
+	lines, readErr := ReadLines(filename)
+	if readErr != nil {
+		utilLog.Error("Unable to read file", "file", filename, "error", err)
```

Comment:
utilLog.Error("Unable to read file", "file", filename, "error", err)
to
utilLog.Error("Unable to read file", "file", filename, "error", readErr)

Reply:
Fixed